### PR TITLE
Sentiment Scale: Accessibility issues

### PR DESCRIPTION
**Pull request by HarieshPugazhendhi**
## Description

This Sentiment Scale component addresses accessibility to include  customizable aria-label attributes for improved screen reader support.

References #
#1976 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- In the Sentiment-scale component added a control option for aria-label in stories and typed a custom label in it. 
- Verified the presence and functionality of the aria-label attributes  in the Storybook environment by manually inspecting the rendered HTML.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by ElishaSamPeterPrabhu — DISMISSED** [review]

**Comment by coliff** [comment]
Good PR - thanks! There is a remaining issue with the Storybook canvas page:

- The buttons here (https://modus-web-components.trimble.com/?path=/docs/user-inputs-select--default) are disabled. We need to remove `disabled="false"` from `modus-sentiment-scale`.
image

- Not a problem, but the `aria-disabled="false"` which is on by default is unnecessary. Can it be removed? ``aria-disabled="true"`` should be set when it's disabled, but if it's not disabled then `aria-disabled` attribute should removed altogether.

File: stencil-workspace/src/components/modus-sentiment-scale/modus-sentiment-scale.tsx [hunk 1]
```diff
@@ -79,7 +83,7 @@ export class ModusSentimentScale {
                 class={containerClass}
                 onClick={() => this.handleSentimentClick(key)}
                 onKeyDown={(event) => this.handleKeyDown(event, key)}>
-                <SentimentIconMap icon={iconName} size={`${this.type === THUMB_SENTIMENT_TYPE ? '32' : '24'}`} />
+                <SentimentIconMap icon={iconName} size={`${this.type === 'thumbs' ? '32' : '24'}`} />
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-sentiment-scale/modus-sentiment-scale.tsx:86]: This should still use the constant `THUMB_SENTIMENT_TYPE` for maintainability.

**Review by cjwinsor — CHANGES_REQUESTED** [review]

File: stencil-workspace/src/components/modus-sentiment-scale/modus-sentiment-scale.tsx (hunk 1 above)
Comment (HarieshPugazhendhi) [reply stencil-workspace/src/components/modus-sentiment-scale/modus-sentiment-scale.tsx:86]: Modified the code to still use the constant for maintainability.

**Comment by cjwinsor** [comment]
@HarieshPugazhendhi Just checking if you meant to push latest changes after that last comment or it was just a confirmation of approach.

**Comment by HarieshPugazhendhi** [comment]
> Good PR - thanks! There is a remaining issue with the Storybook canvas page:
> 
> * The buttons here (https://modus-web-components.trimble.com/?path=/docs/user-inputs-select--default) are disabled. We need to remove `disabled="false"` from `modus-sentiment-scale`. This is important because `disabled="false"` actually just means disabled (i.e. it's the same as `disabled="true"`!)
>   image
> * Not a problem, but the `aria-disabled="false"` which is on by default is unnecessary. Can it be removed? `aria-disabled="true"` should be set when it's disabled, but if it's not disabled then `aria-disabled` attribute should removed altogether.

Hey Coliff,
                The changes that you mentioned to remove disabled="false" from modus-sentiment-scale is complicated. Removing this parameter will affect the component's toggle functionality. Furthermore, other parameters rely on the disabled attribute, making its presence important. So what else could be done either?

**Review by coliff — APPROVED** [review]
The Canvas demo has accessibility issues (it's ALWAYS announced as disabled - even is disabled if set as False)... but the main functionality on Docs tab is working good so I'll approve.

File: stencil-workspace/storybook/stories/components/modus-sentiment-scale/modus-sentiment-scale.stories.tsx [hunk 2]
```diff
@@ -36,9 +44,9 @@ export default {
   },
 };
 
-const Template = ({iconsType,
+const Template = ({ariaLabel,iconsType,
 disabled}) => html`
-<modus-sentiment-scale type="${iconsType}" disabled="${disabled}"></modus-sentiment-scale>
+<modus-sentiment-scale  aria-label=${ariaLabel} type="${iconsType}" disabled="${disabled}"></modus-sentiment-scale>
```
Comment (cjwinsor) [inline stencil-workspace/storybook/stories/components/modus-sentiment-scale/modus-sentiment-scale.stories.tsx:49]: @HarieshPugazhendhi I think you can update this to `?disabled=$"{disabled}"` to satisfy the need to only add the property when it's true.

Here is how attributes are handled in the html templating
https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions

**Review by cjwinsor — APPROVED** [review]

**Pull request merged**